frontend/src/app.js

```diff
@@ -79,16 +87,17 @@ search.addWidgets([
   }),
   // Loading indicator
   {
-    render: ({ searchMetadata = {} }) => {
-      const { isSearchStalled } = searchMetadata;
-      document
-        .getElementById('loading')
-        .classList.toggle('hidden', !isSearchStalled);
-      document
-        .getElementById('stats')
-        .classList.toggle('hidden', isSearchStalled);
-      if (!isSearchStalled)
-        document.getElementById('under-progress').classList.remove('hidden');
+    render: () => {
+      // console.log('searchMetadata', searchMetadata);
+      // const { isSearchStalled } = searchMetadata;
+      // document
+      //   .getElementById('loading')
+      //   .classList.toggle('hidden', !isSearchStalled);
+      // document
+      //   .getElementById('stats')
+      //   .classList.toggle('hidden', isSearchStalled);
+      // if (!isSearchStalled)
+      //   document.getElementById('under-progress').classList.remove('hidden');
```

Comment:
Here two remarks:

1. It is not proper to keep commented code. We have GIT which preserves old version of the code, therefore commented code is just a garbage.
2. This particular task, I believe, does not require any changes to the loading indicator, therefore I consider this change as irrelevant to the task. Please revert.

Reply:
remark 1 reply: it could be a personal preference. i generally don't like garbage code. reason to keep this code commented is just that i din't want to remove code written by someone else.

remark 2 reply: that loader is handled in newly created custom widget. so i removed it from here in order to remove duplicate code.

Reply:
1. I don't agree that it's a matter of preference, sorry. Code written by someone else is always kept in the previous version of the source code in GIT, so you are not removing it altogether. But garbage is garbage. Just imagine if collaborators of a huge project with 100 participants would keep others' code commented instead of just replacing it with own code when needed…
2. This is the point I'm trying to convey in all the comments: it's a wrong path to create a custom `infiniteHits` widget and move there all the logic related to other widgets. Effectively you are making this custom widget empty, dummy.

Reply:
ok, i take my words back :)
i will try to remove commented code at all.

Reply:
see, i agree that this could be a wrong path. and at this point i request you to try to complete this error handling task including second use case where we need to hide error message on next successful request. i am sure that you can do in no time. after so much discussion, you have got some idea about me as a developer. this is the best i can offer. if i continue to collaborate, i can do whatever way you want me to code in.